modules/nf-core/popscle/demuxlet/main.nf

```diff
@@ -0,0 +1,52 @@
+process POPSCLE_DEMUXLET {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/popscle:0.1beta--h2c78cec_0' :
+        'biocontainers/popscle:0.1beta--h2c78cec_0' }"
+
+    input:
+    tuple val(meta), path(plp_input), path(bam), path(donor_genotype)
+
+    output:
+    tuple val(meta), path('*.best'), emit: demuxlet_result
+    path 'versions.yml'            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def input = plp_input ? "--plp ${plp_input}" : "--sam $bam"
+    def VERSION ='0.1'
```

Comment:
Consider adding a comment about manual version changes (the tool cannot print its version?)

Reply:
The tool cannot print its version so i go ahead with hard coding..

Reply:
No problem, but it would be nice to put a comment in the code to remind people about it when they update the module, the docs recommend `// WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.`

Reply:
thanks!